### DISCUSSION TITLE
https://django-crm.micropyramid.com site

### DISCUSSION THREAD

**Discussion by xyzPeter** [Django-CRM/Django-CRM#474] [Q&A] [answered]
Hi,

I'm interested in reviewing this project. I notice that the sample site (https://django-crm.micropyramid.com/) has been down for a couple of days. Are the plans to bring this back online, or is this down permanently?

Thanks.

**Comment by ashwin31** [comment, marked as answer]
It will be up in bottlecrm.com soon. not able to spend much time on this. Hoping to get it up by and of this month in full working condition.
